tests/test_rds2/test_rds2_clusters.py

```diff
@@ -335,3 +335,231 @@ def test_stop_db_cluster_unknown_cluster():
+        DBClusterInstanceClass="db.m1.small",
+        MasterUsername="root",
+        MasterUserPassword="hunter2000",
+        Port=1234,
+    )
+
+    source_snapshot = conn.create_db_cluster_snapshot(
+        DBClusterIdentifier="db-primary-1", DBClusterSnapshotIdentifier="snapshot-1",
+    ).get("DBClusterSnapshot")
+
+    target_snapshot = conn.copy_db_cluster_snapshot(
+        SourceDBClusterSnapshotIdentifier="snapshot-1", TargetDBClusterSnapshotIdentifier="snapshot-2"
+    ).get("DBClusterSnapshot")
+
+    target_snapshot.get("Engine").should.equal("postgres")
+    target_snapshot.get("DBClusterIdentifier").should.equal("db-primary-1")
+    target_snapshot.get("DBClusterSnapshotIdentifier").should.equal("snapshot-2")
+    result = conn.list_tags_for_resource(ResourceName=target_snapshot["DBClusterSnapshotArn"])
+    result["TagList"].should.equal([])
+
+
+@mock_rds2
+def test_describe_db_cluster_snapshots():
+    conn = boto3.client("rds", region_name="us-west-2")
+    conn.create_db_cluster(
+        DBClusterIdentifier="db-primary-1",
+        AllocatedStorage=10,
+        Engine="postgres",
+        DatabaseName="staging-postgres",
+        DBClusterInstanceClass="db.m1.small",
+        MasterUsername="root",
+        MasterUserPassword="hunter2000",
+        Port=1234,
+    )
+
+    created = conn.create_db_cluster_snapshot(
+        DBClusterIdentifier="db-primary-1", DBClusterSnapshotIdentifier="snapshot-1"
+    ).get("DBClusterSnapshot")
+
+    created.get("Engine").should.equal("postgres")
+
+    by_database_id = conn.describe_db_cluster_snapshots(
+        DBClusterIdentifier="db-primary-1"
+    ).get("DBClusterSnapshots")
+    by_snapshot_id = conn.describe_db_cluster_snapshots(DBClusterSnapshotIdentifier="snapshot-1").get(
+        "DBClusterSnapshots"
+    )
+    by_snapshot_id.should.equal(by_database_id)
+
+    snapshot = by_snapshot_id[0]
+    snapshot.should.equal(created)
+    snapshot.get("Engine").should.equal("postgres")
+
+    conn.create_db_cluster_snapshot(
+        DBClusterIdentifier="db-primary-1", DBClusterSnapshotIdentifier="snapshot-2"
+    )
+    snapshots = conn.describe_db_cluster_snapshots(DBClusterIdentifier="db-primary-1").get(
+        "DBClusterSnapshots"
+    )
+    snapshots.should.have.length_of(2)
+
+
+@mock_rds2
+def test_delete_db_cluster_snapshot():
+    conn = boto3.client("rds", region_name="us-west-2")
+    conn.create_db_cluster(
+        DBClusterIdentifier="db-primary-1",
+        AllocatedStorage=10,
+        Engine="postgres",
+        DatabaseName="staging-postgres",
+        DBClusterInstanceClass="db.m1.small",
+        MasterUsername="root",
+        MasterUserPassword="hunter2000",
+        Port=1234,
+    )
+    conn.create_db_cluster_snapshot(
+        DBClusterIdentifier="db-primary-1", DBClusterSnapshotIdentifier="snapshot-1"
+    )
+
+    conn.describe_db_cluster_snapshots(DBClusterSnapshotIdentifier="snapshot-1").get("DBClusterSnapshots")[0]
```

Comment:
```suggestion
    conn.describe_db_cluster_snapshots(DBClusterSnapshotIdentifier="snapshot-1")
```
Just a minor niggle, but I think this is more readable. All we need to know is whether the request succeeds

Reply:
Done